src/compas/files/urdf.py

```diff
@@ -204,8 +200,17 @@ def install_parser(cls, parser_type, *tags):
         ----------
         parser_type : type
             Python class handling URDF parsing of the tag.
-        tags : str
+        *tags : list[str]
             One or more URDF string tag that the parser can parse.
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ValueError
+            If `tags` is empty.
```

Comment:
```suggestion
            If ``tags`` is empty.
```

Reply:
i got a bit confused with these guys.

if i understand correctly, variable names should be \`x\`. only values and code snippets should have \`\`x = 0\`\`